app/assets/javascripts/add-comment.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready(function(){
+  $('.new-comment-form').on("ajax:complete", function(e, data) {
+    comment_list = $(this).closest('.add-new-comment')
+    comment_list.prepend(data.responseText)
```

Comment:
I like that you saved the jquery selection to a variable. Helps document your intent. However, you should always make sure to use the `var` keyword when creating variables (unless you have a good reason not to). Without it, JS is going to start looking up the scope chain for a variable named `comment_list`. When it reaches the top and has not yet found it, it will create a global variable at that point. If you use `var` it will keep it locally scoped.